### PR TITLE
Made it sweeeeter , fixes #16

### LINKED ISSUE #16: sweet-pumpkin-pie
The sweeter version

### PR DESCRIPTION
**Pull request by Sabryr**
Made a sweeter version. With lots of sugar

### CODE REVIEW

File: desserts/sweet-pumpkin-pie.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Take out the inside of the pumpkin
+- Use 2 cups of pumpkin pulp per pie
+
+
+## Ingredients
+
+- Pie shell
+- Nutmeg
+- Can of evaporated milk
+- 40 cup sugar
```
Comment (dhanyapushpadas) [inline desserts/sweet-pumpkin-pie.md:17]: it's too sweet , maybe we could have less sugar.  Thanks for the recipe. its yummy recipe

**Review by dhanyapushpadas — CHANGES_REQUESTED** [review]
4 cups of sugar

File: desserts/sweet-pumpkin-pie.md (hunk 1 above)
Comment (NeMaVo) [reply desserts/sweet-pumpkin-pie.md:17]: I agree. maybe a typo?

Comment (dhanyapushpadas) [inline desserts/sweet-pumpkin-pie.md:17]:
```suggestion
- 4 cup sugar
```

**Review by dhanyapushpadas — APPROVED** [review]

**Comment by dhanyapushpadas** [comment]
Thank you

**Pull request merged**